### ISSUE TITLE
Add Image Size (in MB) Comparison for Images (Similar to Vuln Comparison)

### ISSUE THREAD

**Issue by jspeed-meyers** [chainguard-dev/edu#1461]
**Is your feature request related to a problem? Please describe.**
Yes. Troy Scott asked me if it would be possible to add image size comparison data similar to the vuln count comparison data to Chainguard Academy. In his words:

> I think if we had this in the comparison for each image it would be helpful- that way it’s a one stop shop to compare CVEs, image size, etc

**Describe the solution you'd like**
A visual comparison (could be a graph, could be a table) added to the Chainguard Academy website that compares image size (in MB) between Chainguard Images and comparable images.

**Describe alternatives you've considered**
- Don't do it.
- Simply put this information in a static blog post.
- Simply put this information on the product page somewhere.

**Additional context**

**Comment by jspeed-meyers** [comment]
cc @ltagliaferri @erikaheidi
